pkg/api/controller.go

```diff
@@ -3600,3 +3634,10 @@ func (c *Controller) authorize(w http.ResponseWriter, r *http.Request, perms per
 	}
 	return true
 }
+
+func (c *Controller) isNameValid(name string, nameType string) (bool, string) {
+	if strings.Contains(name, "%") {
+		return false, fmt.Sprintf("%s name cannot contain '%'", nameType)
```

Comment:
(Apart from the govet format string error...)

I think validation belongs in the auth service not in the controller.

Also llease comment here why a name with `%` is invalid.  I remember something about the React router component being broken.  Otherwise it is easy to read this code and assume we have some silly format-string vulnerability or worse.

Reply:
I'm not sure I agree regarding the name validation, as I'd say it's a type of request/input validation. You could make the case for it being in the controller.   

I was also following the example of the existing email address validation, which is placed in the controller.
Regarding the duplicate policy name check, I agree that it's less input validation and more specific to the entity type, so I moved it to auth.

Added comment and fixed string formatting issue.